paz/models/detection/model_util.py

```diff
@@ -0,0 +1,268 @@
+################################################
+### The functions in this file are direct    ###
+### copy of the ones present in the original ###
+### Frustum PointNet work                    ###
+################################################
+
+
+import numpy as np
+import tensorflow as tf
+import os
+import sys
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(BASE_DIR)
+
+# -----------------
+# Global Constants
+# -----------------
+
+NUM_HEADING_BIN = 12
+NUM_SIZE_CLUSTER = 8  # one cluster for each type
+NUM_OBJECT_POINT = 512
+
+g_type2class = {'Car': 0, 'Van': 1, 'Truck': 2, 'Pedestrian': 3,
+                'Person_sitting': 4, 'Cyclist': 5, 'Tram': 6, 'Misc': 7}
+g_class2type = {g_type2class[t]: t for t in g_type2class}
+g_type2onehotclass = {'Car': 0, 'Pedestrian': 1, 'Cyclist': 2}
+g_type_mean_size = {'Car': np.array([3.88311640418, 1.62856739989, 1.52563191462]),
+                    'Van': np.array([5.06763659, 1.9007158, 2.20532825]),
+                    'Truck': np.array([10.13586957, 2.58549199, 3.2520595]),
+                    'Pedestrian': np.array([0.84422524, 0.66068622, 1.76255119]),
+                    'Person_sitting': np.array([0.80057803, 0.5983815, 1.27450867]),
+                    'Cyclist': np.array([1.76282397, 0.59706367, 1.73698127]),
+                    'Tram': np.array([16.17150617, 2.53246914, 3.53079012]),
+                    'Misc': np.array([3.64300781, 1.54298177, 1.92320313])}
+g_mean_size_arr = np.zeros((NUM_SIZE_CLUSTER, 3))  # size clustrs
+for i in range(NUM_SIZE_CLUSTER):
+    g_mean_size_arr[i, :] = g_type_mean_size[g_class2type[i]]
+
+
+map_2d_detector = {
+    1: 3,  # car
+    2: 7,  # pedestrian
+    3: 0,  # animal
+    4: 6,  # other_vehicle
+    5: 2,  # bus
+    6: 5,  # motorcycle
+    7: 8,  # truck
+    8: 4,  # emergency_vehicle
+    9: 1,  # bicycle
+}

```

Comment:
This seems to be data specific. It should be removed and put in the data manager class when there is one and not here where the model lives.